src/main/resources/assets/groovyscript/lang/en_us.lang

```diff
@@ -2118,24 +2118,61 @@ groovyscript.wiki.pyrotech.anvil.description=When using hammer or pickaxe it can
+groovyscript.wiki.pyrotech.bloomery.anvilHit.value=Sets how often the bloom needs to be hit on the anvil
+groovyscript.wiki.pyrotech.bloomery.burnTime.value=Sets the time required for the recipe to complete
+groovyscript.wiki.pyrotech.bloomery.anvilType.value=Sets the tool that needs to be used in anvil
+groovyscript.wiki.pyrotech.bloomery.anvilTiers.value=Sets the allowed anvil tiers bloom allows
+groovyscript.wiki.pyrotech.bloomery.experience.value=Sets the experience given from bloom
+groovyscript.wiki.pyrotech.bloomery.bloomYieldMin.value=Sets minimum amount of drops bloom can give
+groovyscript.wiki.pyrotech.bloomery.bloomYieldMax.value=Sets maximum amount of drops bloom can give
+groovyscript.wiki.pyrotech.bloomery.failureChance.value=Sets the chance for bloom to give failure item instead of the actual output
+groovyscript.wiki.pyrotech.bloomery.failureOutput.value=Sets the output to drop on failure
+
+groovyscript.wiki.pyrotech.brick_crucible.title=Refractory Crucible
+groovyscript.wiki.pyrotech.brick_crucible.description=Melts an item into liquid.
+groovyscript.wiki.pyrotech.brick_crucible.burnTime.value=Sets the time required for the recipe to complete
+groovyscript.wiki.pyrotech.brick_crucible.add=Adds recipes in the format `name`, `input`, `fluidOutput`, `burnTime`
+groovyscript.wiki.pyrotech.brick_crucible.note0=Can inherit recipes from [Stone Crucible](./stone_crucible.md)
+
 groovyscript.wiki.pyrotech.brick_kiln.title=Refractory Kiln
 groovyscript.wiki.pyrotech.brick_kiln.description=Converts an item into a new one by burning it. Has a chance to fail
 groovyscript.wiki.pyrotech.brick_kiln.burnTime.value=Sets the time required for the recipe to complete
 groovyscript.wiki.pyrotech.brick_kiln.failureChance.value=Sets the chance to fail the recipe
 groovyscript.wiki.pyrotech.brick_kiln.failureOutput.value=Sets the output when the recipe failed
 groovyscript.wiki.pyrotech.brick_kiln.add=Adds recipes in the format `name`, `input`, `output`, `burnTime`, `failureChance`, `failureOutput`
+groovyscript.wiki.pyrotech.brick_kiln.note0=Can inherit recipes from [Pit Kiln](./pit_kiln.md) and [Stone Kiln](./stone_kiln.md)
 
 groovyscript.wiki.pyrotech.brick_oven.title=Refractory Oven
 groovyscript.wiki.pyrotech.brick_oven.description=When powered by burning fuel can convert items
 groovyscript.wiki.pyrotech.brick_oven.duration.value=Sets the time required for the recipe to complete
 groovyscript.wiki.pyrotech.brick_oven.add=Adds recipes in the format `name`, `input`, `output`, `duration`
+groovyscript.wiki.pyrotech.brick_oven.note0=Can inherit recipes from [Crude Drying Rack](./crude_drying_rack.md), [Drying Rack](./drying_rack.md) and [Stone Oven](./stone_oven.md)
+
+groovyscript.wiki.pyrotech.brick_sawmill.title=Refractory Sawmill
+groovyscript.wiki.pyrotech.brick_sawmill.description=Converts an item, likely wood, by cutting it to a smaller item and drops wood chips.
```

Comment:
in general, i think the descriptions should be primarily representative of what the process itself is, not what the process is used for - as an example, for the mekanism sawmill i say it `Converts an input itemstack into an output itemstack, with an optional additional output.`, and not `turns logs into planks`.

so in this case, i think i would say something like `Converts an input itemstack into an output itemstack after a duration, with an additional catalyst itemstack that may lose durability and the ability to output a configurable amount of woodchips.`